src/hdkey.ts

```diff
@@ -20,22 +19,24 @@ export default class EthereumHDKey {
     return new EthereumHDKey(HDKey.fromExtendedKey(base58Key))
   }
 
-  constructor(private readonly _hdkey?: any) {}
+  constructor(private readonly _hdkey: HDKey) {}
 
   /**
    * Returns a BIP32 extended private key (xprv)
    */
-  public privateExtendedKey(): Buffer {
+  public privateExtendedKey(): string {
```

Comment:
This is also a breaking change, together with the corresponding public* method. What is the reasoning?

Reply:
console.log()-ging this on 1.0.2 outputs: `xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY`

The type was incorrect.

Reply:
👍